packages/gateway/src/offchainexample.eth.json

```diff
@@ -0,0 +1,20 @@
+{
+    "offchainexample.eth": {
+        "addresses": {
```

Comment:
The address and text records should probably be different between the wildcard and regular record, so people can verify which one they're getting.